src/main/resources/configCivcraft.yml

```diff
@@ -9487,7 +9472,7 @@ recipes:
     input:
       emeralds:
         material: EMERALD
-        amount: 5
+        amount: 10
```

Comment:
You're not wrong @walkersgaming just don't see the point of making _increases_ in a PR devoted to _decreases_, and if it's cheap either way, leave it alone. :shrug: